policyengine_us/parameters/gov/states/nm/tax/rebates/low_income_rebate/amount/five_exemptions.yaml

```diff
@@ -0,0 +1,94 @@
+description: New Mexico provides the following low income comprehensive tax rebate for filers with five exemptions.
+brackets:
+  - threshold:
+      2021-01-01: 0
+    amount:
+      2021-01-01: 455
+  - threshold: 
+      2021-01-01: 1_001
+    amount:
+      2021-01-01: 570
+  - threshold:
+      2021-01-01: 7_501
+    amount:
+      2021-01-01: 575
+  - threshold: 
+      2021-01-01: 9_001
+    amount: 
+      2021-01-01: 510
+  - threshold: 
+      2021-01-01: 10_001
+    amount: 
+      2021-01-01: 445
+  - threshold: 
+      2021-01-01: 11_501
+    amount: 
+      2021-01-01: 365
+  - threshold: 
+      2021-01-01: 13_001
+    amount: 
+      2021-01-01: 315
+  - threshold: 
+      2021-01-01: 14_501
+    amount: 
+      2021-01-01: 285
+  - threshold: 
+      2021-01-01: 16_501
+    amount: 
+      2021-01-01: 250
+  - threshold: 
+      2021-01-01: 18_001
+    amount: 
+      2021-01-01: 220
+  - threshold: 
+      2021-01-01: 19_501
+    amount: 
+      2021-01-01: 185
+  - threshold: 
+      2021-01-01: 23_001
+    amount: 
+      2021-01-01: 170
+  - threshold: 
+      2021-01-01: 24_501
+    amount: 
+      2021-01-01: 155
+  - threshold: 
+      2021-01-01: 26_001
+    amount: 
+      2021-01-01: 140
+  - threshold: 
+      2021-01-01: 27_501
+    amount: 
+      2021-01-01: 130
+  - threshold: 
+      2021-01-01: 29_501
+    amount: 
+      2021-01-01: 115
+  - threshold: 
+      2021-01-01: 31_001
+    amount: 
+      2021-01-01: 100
+  - threshold: 
+      2021-01-01: 32_501
+    amount: 
+      2021-01-01: 80
+  - threshold: 
+      2021-01-01: 34_001
+    amount: 
+      2021-01-01: 65
+  - threshold: 
+      2021-01-01: 36_001
+    amount: 
+      2021-01-01: 0
+metadata:
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  type: single_amount
+  label: New Mexico comprehensive low income tax rebate five exemptions amount
+  reference:
+    - title: New Mexico 2022 PIT-RC, page 58, SECTION II LOW INCOME COMPREHENSIVE TAX REBATE
+      href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/1afc56af-ea90-4d48-82e5-1f9aeb43255a/PITbook2022.pdf#page=58
```

Comment:
could you add one [legal code ](https://casetext.com/regulation/new-mexico-administrative-code/title-3-taxation/chapter-3-personal-income-taxes)reference

Reply:
all yaml files